src/components/templates/PartyMap/components/Map/MapRoom.scss

```diff
@@ -41,6 +41,7 @@ $label-visible: flex;
 
   &--unclickable {
     cursor: default;
+    z-index: revert;
```

Comment:
Is `revert` even a valid value here? I can't see anything about it when googling?

- https://developer.mozilla.org/en-US/docs/Web/CSS/z-index

Reply:
Not sure. It was suggested by the IDE. The intended outcome is to turn the z-index to a default

Reply:
By my read of the above link, setting it to `auto` should probably do that.

---

Though searching for `revert` on it's own lead me to this:

- https://developer.mozilla.org/en-US/docs/Web/CSS/revert
  - > The revert CSS keyword reverts the cascaded value of the property from its current value to the value the property would have had if no changes had been made by the current style origin to the current element. Thus, it resets the property to its inherited value if it inherits from its parent or to the default value established by the user agent's stylesheet (or by user styles, if any exist). It can be applied to any CSS property, including the CSS shorthand all.
  - > The `revert` keyword works exactly the same as `unset` in many cases. The only difference is for properties that have values set by the browser or by custom stylesheets created by users (set on the browser side).
  - > The `revert` keyword is different from and should not be confused with `initial`, which uses the initial value defined on a per-property basis by the CSS specifications. In contrast, user-agent stylesheets set default values on the basis of CSS selectors.
  
If we weren't using BEM (as I don't technically think we have a parent in this selector path, as `&--unclickable` will just turn into `.maproom--unclickable` rather than `.maproom.maproom--unclickable`), then this would probably apply:

- https://developer.mozilla.org/en-US/docs/Web/CSS/revert#revert_on_a_parent
  - > Reverting effectively removes the value for the element you select with some rule and only for that element.

I think if we moved the z-index `revert` part to a new block that looked something like this, then it would probably work in this 'revert on a parent' way:

```scss
&&--unclickable {
  z-index: revert;
}
```

Which should then output the `.maproom.maproom--unclickable` selecter I believe.

---
  
(sidenote: also opened https://github.com/mdn/content/issues/3047 to fix the `z-index` docs on this)

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1225/commits/7db0a2f720d51d4000e5f94ebcb251924cec7345

Reply:
Didn't want to follow the final part of my suggestion there then?

> I think if we moved the z-index revert part to a new block that looked something like this, then it would probably work in this 'revert on a parent' way:
> 
> ```scss
> &&--unclickable {
>   z-index: revert;
> }
> ```

> Which should then output the .maproom.maproom--unclickable selector with this revert I believe.

Reply:
The implementation with the `auto` seems more easy-to-understand and obvious IMO

Reply:
Mm, fair enough. I don't believe it will always strictly be accurate. But it's probably 'good enough'.

Reply:
I believe the edge case would be, as an example, would be if a parent was set to say `z-index: 100`, the room was set to `z-index: 150`, and the unclickable room is set to `z-index: auto`.

The expectation would be that it 'falls back to `100`, but `auto` will set it to (essentially) 'nothing'.

I think.

Reply:
I don't think I would expect it to be 100. When I used it in my case, my intention was to disable `z-index`, not to lower the value/set the parent's value.

Reply:
Fair. Either way, good enough for this PR.